build/metadata/github-actions/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM launchdarkly/ld-find-code-refs-github-action:x.x.x
```

Comment:
Is this file actually used anywhere? I think we have the file we use in the root of the repo as `Dockerfile.github` right?

Reply:
Yep this file is required by the github runtime to find our docker image. The other file you mentioned `Dockerfile.github` is used internally by us with goreleaser to publish to dockerhub. So they are distinct in purpose and they are different files.